src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -60,6 +60,138 @@ using var meterProvider = Sdk.CreateMeterProviderBuilder()
+|--------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------|-------------------|------------|------------------|------------------|
+| process.runtime.dotnet.**monitor.lock_contention.count**     | The number of times there was contention when trying to acquire a monitor lock since the process start. Monitor locks are commonly acquired by using the lock keyword in C#, or by calling Monitor.Enter() and Monitor.TryEnter().                                                  | `{times}`   | ObservableCounter | `Int64`    |                  |                  |
+| process.runtime.dotnet.**thread_pool.threads.count**         | The number of thread pool threads that currently exist.                                                                                                                                                                                                                             | `{threads}` | ObservableGauge   | `Int32`    |                  |                  |
+| process.runtime.dotnet.**thread_pool.completed_items.count** | The number of work items that have been processed by the thread pool since the process start.                                                                                                                                                                                       | `{items}`   | ObservableCounter | `Int64`    |                  |                  |
+| process.runtime.dotnet.**thread_pool.queue.length**          | The number of work items that are currently queued to be processed by the thread pool.                                                                                                                                                                                              | `{items}`   | ObservableGauge   | `Int64`    |                  |                  |
+| process.runtime.dotnet.**timer.count**                       | The number of timer instances that are currently active. Timers can be created by many sources such as System.Threading.Timer, Task.Delay, or the timeout in a CancellationSource. An active timer is registered to tick at some point in the future and has not yet been canceled. | `{timers}`  | ObservableGauge   | `Int64`    |                  |                  |
+
+The APIs used to retrieve the values are:
+
+* [Monitor.LockContentionCount](https://docs.microsoft.com/dotnet/api/system.threading.monitor.lockcontentioncount?view=netcore-3.1):
```

Comment:
Was it intentional to link the .NET Core 3.1 version instead of the current NET 6.0?

Reply:
Maybe we should omit the version/view parameter, so that always the latest version will be shown.

Reply:
It was intentional, but I'll remove the `?view=*` part, as showing the latest makes more sense.

I intentionally picked to link to .NET Core 3.1 as it is the earliest supported platform. (as shown in "These metrics are only available when targeting .NET Core 3.1 or later.") Some APIs are only available in .NET 6 and later and I picked the link for .NET 6 for them as well. On second thought, removing it also makes sense and that way it's the reader's responsibility to mind which versions are supported.